recipes/libcusolver/meta.yaml

```
@@ -0,0 +1,104 @@
+{% set name = "libcusolver" %}
+{% set version = "11.4.2.57" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[0] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c9460db500d0c7d14debb2c8e0d88fc26a39197363f1b000ccf760866f636f12  # [linux64]
+  sha256: bf3c9a090652d011aafe2190c6f26fe108194cf8512d3c571e406054740168bb  # [ppc64le]
+  sha256: 0ca46b73520657f96f921711c05af7053bd398328d510aecddfff2521c000e91  # [aarch64]
+  sha256: c09836df1f43a3496f2b61ad8827af662ace298508869564a40c30d909ba550f  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+
+outputs:
+  - name: libcusolver
+    requirements:
+      run:
+        - libcublas =12.*
+        - libnvjitlink =12.*
+    files:
+      - lib/libcusolver*.so.*            # [linux]
+      - targets/*/lib/libcusolver*.so.*  # [linux]
+      - Library\bin                      # [win]
+    test:
+       commands:
+         - test -f $PREFIX/lib/libcusolver.so  # [unix]
+         - test -f $PREFIX/lib/libcusolverMg.so  # [unix]
+         - if not exist %LIBRARY_LIB%\\cusolver.lib exit 1  # [win]
+         - if not exist %LIBRARY_LIB%\\cusolverMg.lib exit 1  # [win]
+
+  - name: libcusolver-dev
+    requirements:
+      run:
+        - libcusolver >={{ version }}
```

@Bdice mentioned to me that this is going to have trouble (ex: installing an old header + new & incompatible binary), so I think this is safer:
```suggestion
        - {{ pin_subpackage("libcusolver", max_pin="x") }}
```

---

It seems [cudart](https://github.com/conda-forge/staged-recipes/pull/21723/files#diff-0dd2e318a324e35cf19682e69f69f441a7bbb76fd01f33c3615ac69a66c6f4b3R51) is putting it under `run_exports`, I'll follow.